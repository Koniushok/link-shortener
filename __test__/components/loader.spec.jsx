import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Loader from '../../src/components/loader';

describe('Loader component', () => {
  describe('Loader component initial', () => {
    it('render correctly component', () => {
      const output = shallow(<Loader />);
      expect(shallowToJson(output)).toMatchSnapshot();
    });
  });
});
