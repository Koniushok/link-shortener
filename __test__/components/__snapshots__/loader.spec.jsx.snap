// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component Loader component initial render correctly component 1`] = `
<styled.div>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</styled.div>
`;
